packages/mdc-tab-scroller/README.md

```diff
@@ -0,0 +1,72 @@
+<!--docs:
+title: "Tab Scroller"
+layout: detail
+section: components
+excerpt: "Tab Scroller allows for smoothing animated scrolling of tab content"
+iconId: tab
+path: /catalog/tab-scroller/
+-->
+
+# Tab Scroller
+
+<!--<div class="article__asset">
+  <a class="article__asset-link"
+     href="https://material-components-web.appspot.com/tab-scroller.html">
+    <img src="{{ site.rootpath }}/images/mdc_web_screenshots/tab-scroller.png" width="363" alt="Tab scroller screenshot">
+  </a>
+</div>-->
+
+Tab Scroller allows for smoothing animated scrolling of tab content
+
+## Installation
+```
+npm install --save @material/tab-scroller
```

Comment:
Drop `--save` from this (and other tab readmes if necessary). (I dropped this from our other readmes a while ago)